compose-dev.yml

```diff
@@ -10,8 +10,8 @@ master:
   net: host
   environment:
     MESOS_ZK: zk://localhost:2181/mesos
-    MESOS_HOSTNAME: localhost
-    MESOS_IP: 127.0.0.1
+    MESOS_HOSTNAME: 192.168.99.100
```

Comment:
Have a fix for this in https://github.com/HubSpot/Singularity/pull/1655 , I'll merge through to master so you can pull it in